### DISCUSSION TITLE
求快速获取菲林！😭

### DISCUSSION THREAD

**Discussion by Lbybbs** [share121/inter-knot#2109] [其他]
有没有快速获取菲林的方法啊，44级找不到地方赚菲林了😭，很急我要抽限定😭
![06a14c948cb2f3f7bc512052de2a6f59_6743737367233673559](https://github.com/user-attachments/assets/ccfd87c3-e78c-41e4-878f-42a6e6e923bd)

**Comment by ENDer-Z1** [comment]
选择蓝绿修改器，享传奇绳匠人生

**Reply by Lbybbs** [reply]
别了，我要做合法绳网市民！……唔…是合法新艾利都公民！

**Reply by qwer729** [reply]
蓝绿修改器是微信和支付宝，也就是氪

**Comment by hfwang132** [comment]
打零工赚钱，然后就可以充值了

**Comment by yue0ying** [comment]
蓝绿修改器，你值得信赖

**Comment by HFOxin** [comment]
我们可以靠肝，先肝一下上海，上海不愧是大城市，随便肝了两分钟就有6480菲林了

**Comment by YuTinMIn** [comment]
答：你可以做活动、做任务、打零号空洞（废话）

**Comment by LuAn-afl** [comment]
他们说的都不靠谱我建议上我号我号好多任务都没做主线才推到第二章所以上我号肝就能得到很多菲琳了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by Aha-SWYH** [comment]
拍一张录像店兄妹的图片，然后说法厄同被抓了 需要多少菲林

**Comment by LIU2-GIT** [comment]
蓝绿修改器<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
